optional/e2e/features/footer.feature

```diff
@@ -0,0 +1,53 @@
+#language: fr
+
+@footer
+Fonctionnalité: Footer
+  Pour en savoir plus sur le site Code du travail numérique
+  En tant que visiteur
+  Je veux pouvoir consulter les fonctionnalités et pages présentes dans le footer
+
+  Scénario: Pages statiques
+    Soit un utilisateur sur la page d'accueil
+
+    Alors je vois "Besoin de plus d’information"
+    Quand je clique sur "Contacter nos services en région"
+    Alors je vois "Contact téléphonique"
+    Quand je cherche "75" dans le champ "search-service"
+    Alors le lien "http://idf.direccte.gouv.fr" pointe sur "http://idf.direccte.gouv.fr"
+    Quand je ferme la modale
+
+    Quand je clique sur "Le droit du travail"
+    Alors je vois "Qu’est-ce que le droit du travail"
+    Alors je vois "Quels sont les textes à l’origine du droit du travail"
+    Alors je vois "Existe-t-il une hiérarchie entre les textes"
+
+    Quand je clique sur "Glossaire"
```

Comment:
I would move that to a dedicated one where we can test glossaire tooltip too

Reply:
J'attend que nico refasse les tooltip avant d'aller plus loin